context/joke-context.tsx

```
@@ -1,56 +1,280 @@
+        setLoading({ isLoading: false, error: null })
+        return categories
+      } catch (error) {
+        const errorMessage = error instanceof Error ? error.message : 'Unknown error'
+        setLoading({ isLoading: false, error: errorMessage })
+        return []
+      }
+    }
+
+    async function getJokeCountByCategory(category: string): Promise<number> {
+      const supabase = createClient()
+      const { count, error } = await supabase
```

ESLint

[error] 'count' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)